ports/openssl/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "openssl",
-  "version": "3.1.4",
+  "version": "3.2.0",
   "port-version": 1,
```

Please refer to the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md#versioning) on how to properly update `port-version`.

---

I guess this was an oversight.
```suggestion
  "version": "3.2.0",
```